modules/vaos/app/services/vaos/appointment_service.rb

```diff
@@ -24,6 +24,12 @@ def post_appointment(request_object_body)
           data: OpenStruct.new(response.body),
           meta: {}
         }
+      rescue Common::Exceptions::BackendServiceException => e
+        # TODO: Reevaluate the need to log clinic data three months after launch (6/15/20)
+        if e.key == 'VAOS_400'
+          Rails.logger.warn('Clinic does not support VAOS appointment create', clinic: params['clinic'])
```

Comment:
Do we need to log the facility id, too? I don't think clinic ids are unique across sites.

Reply:
might be able to put some of those in as semantic logging elements / aka tags. Easier to query for later in cloud watch.

Reply:
an example of the clinic hash is:
```ruby
{"site_code"=>"983", "clinic_id"=>"308", "clinic_name"=>"CHY PC KILPATRICK", "clinic_friendly_location_name"=>"Green Team Clinic1", "institution_name"=>"CHYSHR-Cheyenne VA Medical Center", "institution_code"=>"983"}
```
I think `site_code` === `facility_id` right? If I pull that out then both create and cancel log lines would be the same.

Reply:
Yeah, I think site_code is the right facility id in this case.